client/my-sites/checkout/checkout/credit-card-payment-box.jsx

```diff
@@ -173,12 +183,12 @@ var CreditCardPaymentBox = React.createClass( {
 		return (
 			<PaymentBox
 				classSet="credit-card-payment-box"
-				title={ this.translate( 'Secure Payment' ) }>
+				title={ this.props.translate( 'Secure Payment' ) }>
 				{ this.content() }
 			</PaymentBox>
 		);
 	}
 
 } );
 
-module.exports = CreditCardPaymentBox;
+module.exports = localize( CreditCardPaymentBox );
```

Comment:
That's enforced by a pre-commit hook - `wpcalypso/i18n-no-this-translate`, see https://git.io/vSwRi . The goal is to move away from mixins iirc.